update.py

```diff
@@ -193,17 +194,20 @@ def fetchlogmessages(site=None, cache=None):
 
 
 def build_one(wiki, fast):
-    '''build one wiki'''
-    debug('Using make for sphinx: %s' % wiki)
-    if platform.system() == "Windows":
-        # This will fail if there's no folder to clean, so no check_call here
-        if not fast:
-            subprocess.run(["make.bat", "clean"], cwd=wiki, shell=True)
-        subprocess.check_call(["make.bat", "html"], cwd=wiki, shell=True)
-    else:
-        if not fast:
-            subprocess.check_call(["nice", "make", "clean"], cwd=wiki)
-        subprocess.check_call(["nice", "make", "html"], cwd=wiki)
+    """build one wiki"""
+    print('Using sphinx-build for sphinx: %s' % wiki)
+
+    source_dir = os.path.join(wiki, 'source')
```

Comment:
pathlib is preferred over os.path.join by many python3 devs due to increased readability with the `/` operator on pathlib.PurePath objects.

Reply:
delay to a py3 update